float64 larg_retangulo_pista
float64 alt_retangulo_pista
float64 qtde_contornos_pista
float64 coordenada_X_centro_contorno_pista
float64 coordenada_Y_centro_contorno_pista
float64 qtde_contornos_pista_inicio
float64 coordenada_X_centro_contorno_pista_inicio
float64 coordenada_Y_centro_contorno_pista_inicio
float64 qtde_contornos_pista_final
float64 coordenada_X_centro_contorno_pista_final
float64 coordenada_Y_centro_contorno_pista_final
bool sensor_verde_encontrado
bool sensor_vermelho_encontrado
